testing/load-aware/command_args.yaml

```diff
@@ -36,3 +46,40 @@ utils build_push_image/make:
   image_local_name: coreutils
   tag: make
   dockerfile_path: testing/load-aware/images/Containerfile.make
+
+cluster preload_image/deps:
+  namespace: {{ load_aware.scale_test.namespace }}
+  name: deps
+  image: {{ scale_test_imagestream }}:deps
+  <<: *sutest_preload_image
+
+cluster preload_image/make:
+  namespace: {{ load_aware.scale_test.namespace }}
+  name: make
+  image: {{ scale_test_imagestream }}:make
+  <<: *sutest_preload_image
+
+cluster preload_image/sleep:
+  namespace: {{ load_aware.scale_test.namespace }}
+  name: sleep
+  image: fedora:latest
```

Comment:
can you use `registry.access.redhat.com/ubi8/ubi` ? this image should be already part of the cluster

and, just for information, using an image `shortname` *address* is (now) discouraged,
Docker made it the usual to use eg `fedora:latest`, which turned into `docker.io/lib/fedora:latest`
but that's making you think that `docker.io` is the only image registry!

```
podman pull fedora:latest
Resolved "fedora" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
Trying to pull registry.fedoraproject.org/fedora:latest...
Getting image source signatures
Copying blob ad5077952f52 [========>-----------------------------] 17.4MiB / 70.1MiB
```

so for `fedora`, the correct image value is: `registry.fedoraproject.org/fedora:latest`

Reply:
(and when you update to `ubi`, please also update the comment somewhere else in the PR)

Reply:
Makes sense, I'll switch over to `ubi`. Should I ensure it's part of the cluster by preloading it still?

> (and when you update to ubi, please also update the comment somewhere else in the PR)
Not entirely sure what you mean?

Reply:
> Should I ensure it's part of the cluster by preloading it still?

yes, for consistency, keep the preload code 


>>    (and when you update to ubi, please also update the comment somewhere else in the PR)

> Not entirely sure what you mean?

I'll add another comment to show you ;-)